app/views/puppetclasses/_classes_parameters.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= alert :header => '', :class => 'alert-warning', :close => false,
+          :text => _("Puppet ENC have been extracted to a plugin and will be removed from core in 3.0") %>
```

Comment:
We should probably add a warning in https://github.com/theforeman/foreman/blob/develop/app/helpers/puppet_related_helper.rb#L70 as well so it is also displayed on the puppet classes tab in the form

Reply:
@shiramax could we address this?